pkg/builder/AppDetailBuilder.go

```diff
@@ -384,6 +384,130 @@ func GetDeploymentDetail(request *client.DeploymentDetailRequest) (*client.Deplo
 	return resp, nil
 }
 
+func InstallRelease(request *client.InstallReleaseRequest) (*client.InstallReleaseResponse, error) {
+	releaseIdentifier := request.ReleaseIdentifier
+	conf, err := k8sUtils.GetRestConfig(releaseIdentifier.ClusterConfig)
+	if err != nil {
+		return nil, err
+	}
+	opt := &helmClient.RestConfClientOptions{
+		Options: &helmClient.Options{
+			Namespace: releaseIdentifier.ReleaseNamespace,
+		},
+		RestConfig: conf,
+	}
+
+	helmClientObj, err := helmClient.NewClientFromRestConf(opt)
+	if err != nil {
+		return nil, err
+	}
+
+	// Add or update chart repo starts
+	chartRepoRequest := request.ChartRepository
+	chartRepoName := chartRepoRequest.Name
+	chartRepo := repo.Entry{
+		Name:     chartRepoName,
+		URL:      chartRepoRequest.Url,
+		Username: chartRepoRequest.Username,
+		Password: chartRepoRequest.Password,
+		// Since helm 3.6.1 it is necessary to pass 'PassCredentialsAll = true'.
+		PassCredentialsAll:    true,
+		InsecureSkipTLSverify: true,
+	}
+
+	err = helmClientObj.AddOrUpdateChartRepo(chartRepo)
+	if err != nil {
+		return nil, err
+	}
+	// Add or update chart repo ends
+
+	// Install release starts
+	chartSpec := &helmClient.ChartSpec{
+		ReleaseName:      releaseIdentifier.ReleaseName,
+		Namespace:        releaseIdentifier.ReleaseNamespace,
+		ValuesYaml:       request.ValuesYaml,
+		ChartName:        chartRepoName + "/" + request.ChartName,
```

Comment:
use fmtf instead of string concatenation

Reply:
done